toolbox/include/base/BlockInformation.h

```diff
@@ -68,10 +70,12 @@ class wbt::BlockInformation {
      * @param [out] option implementation-specific block option
      * @return true if the option has been converted. False otherwise
      */
-    virtual bool optionFromKey(const std::string& key, Data &option) const;
+    virtual bool optionFromKey(const std::string& key, double& option) const;
```

Comment:
Why assuming that options can be only double?

Reply:
This was the first step of deprecating the `Data` class. "Was" because I forgot it is used also for `Signal`s. Internally in any case the value is saved in a double format, even if it is a bool.

After this new WB-Toolbox version be properly tested, I'd like to improve `Data` as described [here](https://github.com/robotology/WB-Toolbox/issues/60). Now there are too many changes to test, and I'm postponing activities.